### PR TITLE
EZP-31218: Restyled login screen

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31218
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Added new login image, logo, linear gradient and hyperlink color on the login site

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by lserwatka** [comment]
Could you add a screenshot as well?

**Comment by lucasOsti** [comment]
> Could you add a screenshot as well?

login

File: src/bundle/Resources/views/themes/admin/account/base.html.twig [hunk 1]
```diff
@@ -3,7 +3,9 @@
     <head>
         <meta charset="UTF-8" />
         <title>eZ Platform</title>
-        {{ encore_entry_link_tags('ezplatform-admin-ui-security-base-css', null, 'ezplatform') }}
+        {{
+            encore_entry_link_tags('ezplatform-admin-ui-security-base-css', null, 'ezplatform')
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/account/base.html.twig]: I think it's better in one line

File: src/bundle/Resources/views/themes/admin/account/base.html.twig [hunk 2]
```diff
@@ -13,19 +15,16 @@
...
                     </div>
                     <div class="ez-login__footer">
-                        <span>{{ 'base.footer_copyright'|trans({
-                                '%ez_link_open%': '<a href="http://ez.no">',
-                                '%ez_link_close%': '</a>',
-                                '%current_year%': "now"|date("Y")
-                            })
-                            |desc('eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.')|raw }}
+                        <span
+                            >{{ 'base.footer_copyright'|trans({ '%ez_link_open%': '<a class="ez-login--hyperlink" href="http://ez.no"
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/account/base.html.twig]: This code styling is not very readable, previous was better.

File: src/bundle/Resources/views/themes/admin/account/login/index.html.twig [hunk 3]
```diff
@@ -20,5 +20,5 @@
             <button type="submit" class="btn btn-primary ez-btn ez-btn--login">{{ 'authentication.submit'|trans|desc('Login') }}</button>
         </fieldset>
     </form>
-    <a class="mt-3 text-right" href="{{ path('ezplatform.user.forgot_password') }}">{{ 'authentication.forgot_password'|trans|desc('Forgot password?') }}</a>
+    <a class="mt-3 text-right ez-login--hyperlink" href="{{ path('ezplatform.user.forgot_password') }}">{{ 'authentication.forgot_password'|trans|desc('Forgot password?') }}</a>
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/account/login/index.html.twig]: Wrong BEM usage - `ez-login__hyperlink`

File: src/bundle/Resources/views/themes/admin/account/reset_password/confirmation_page.html.twig [hunk 4]
```diff
@@ -1,9 +1,8 @@
...
-    <p>
-        {{ 'ezplatform.reset_user_password.success'|trans({ '%login%': path('login') })
-        |desc('Your password has been changed. You can <a href="%login%">log in</a> to eZ Platform.')|raw }}
-    </p>
-{%- endblock content -%}
+<h2 class="ez-login__header">{{ 'ezplatform.reset_user_password.change_password'|trans|desc('Change password') }}</h2>
+<p>
+	{{ 'ezplatform.reset_user_password.success'|trans({ '%login%': path('login') })
+        |desc('Your password has been changed. You can <a class="ez-login--hyperlink" href="%login%">log in</a> to eZ Platform.')|raw }}
+</p>{%- endblock content -%}
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/account/reset_password/confirmation_page.html.twig]: Previous code style was better

File: src/bundle/Resources/views/themes/admin/account/reset_password/invalid_link_page.html.twig [hunk 5]
```diff
@@ -1,9 +1,8 @@
...
-    <p>
-        {{ 'ezplatform.reset_user_password.link_is_not_valid'|trans({ '%forgot_user_password%': path('ezplatform.user.forgot_password')})|raw
-        |desc('This link is not valid anymore, you can <a href="%forgot_user_password%">generate a new link</a> to reset your password.') }}
-    </p>
-{%- endblock content -%}
+<h2 class="ez-login__header">{{ 'ezplatform.reset_user_password.change_password'|trans|desc('Change password') }}</h2>
+<p>
+	{{ 'ezplatform.reset_user_password.link_is_not_valid'|trans({ '%forgot_user_password%': path('ezplatform.user.forgot_password')})|raw
+        |desc('This link is not valid anymore, you can <a class="ez-login--hyperlink" href="%forgot_user_password%">generate a new link</a> to reset your password.') }}
+</p>{%- endblock content -%}
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/account/reset_password/invalid_link_page.html.twig]: Previous code style was better.

File: src/bundle/Resources/views/themes/admin/account/reset_password/confirmation_page.html.twig [hunk 6]
```diff
@@ -4,6 +4,6 @@
     <h2 class="ez-login__header">{{ 'ezplatform.reset_user_password.change_password'|trans|desc('Change password') }}</h2>
     <p>
         {{ 'ezplatform.reset_user_password.success'|trans({ '%login%': path('login') })
-        |desc('Your password has been changed. You can <a href="%login%">log in</a> to eZ Platform.')|raw }}
-    </p>
+        |desc('Your password has been changed. You can <a class="ez-login__hyperlink" href="%login%">log in</a> to eZ Platform.')|raw }}
+     </p>
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/account/reset_password/confirmation_page.html.twig]:
```suggestion
    </p>
```

File: src/bundle/Resources/public/scss/_custom.scss [hunk 7]
```diff
@@ -96,7 +96,7 @@ $colors: (
     indigo: $indigo,
     purple: $purple,
     pink: $pink,
-    cyan: $cyan
+    cyan: $cyan,
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_custom.scss]: Please remove `,`

File: src/bundle/Resources/public/scss/_custom.scss [hunk 8]
```diff
@@ -107,7 +107,7 @@ $theme-colors: (
     danger: $ez-color-danger,
     light: $ez-ground-base-pale,
     dark: $ez-color-base-dark,
-    info: $cyan
+    info: $cyan,
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_custom.scss]: Please remove `,`

File: src/bundle/Resources/public/scss/_login.scss [hunk 9]
```diff
@@ -11,13 +11,17 @@
         color: $ez-color-warning-dark;
     }
 
+    &__hyperlink {
+        color: #0005a6;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_login.scss]: I'm not sure, shouldn't we use it by variable? @dew326

File: src/bundle/Resources/views/themes/admin/account/base.html.twig [hunk 10]
```diff
@@ -37,11 +34,11 @@
         <script src="{{ asset('bundles/ezplatformadminuiassets/vendors/bootstrap/dist/js/bootstrap.min.js') }}"></script>
 
         {% if app.request.locale == 'ach_UG' %}
-            <script type="text/javascript">
-                var _jipt = [];
-                _jipt.push(['project', 'ezplatform']);
-            </script>
-            <script type="text/javascript" src="//cdn.crowdin.com/jipt/jipt.js"></script>
+        <script type="text/javascript">
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/account/base.html.twig]: Some strange change, in my opinion these tabs should stay here :thinking:

File: src/bundle/Resources/public/scss/_login.scss (hunk 9 above)
Comment (dew326) [reply src/bundle/Resources/public/scss/_login.scss]: This is an acceptable exception. This color is specific only for this view.

**Comment by inakijv** [comment]
@katarzynazawada you are right. According to [current brand guidelines](https://ez.no/About-eZ/Brand).

**Comment by lserwatka** [comment]
We need to merge this one for next beta.

**Comment by katarzynazawada** [comment]
@lucasOsti, there is still orange outline on Chrome and Edge when clicking button
Screenshot 2019-12-20 at 09 14 09 
Screenshot 2019-12-20 at 09 13 58

**Comment by lserwatka** [comment]
@katarzynazawada is this ready for merge?

**Comment by lserwatka** [comment]
Thank you all! We will ship it 👍

**Pull request merged**